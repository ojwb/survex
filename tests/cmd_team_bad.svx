; pos=no warn=6
*begin survey
*team Wookey note
*team "Olly Betts" instruments tap
; We deliberately skip the line after a bad role to avoid triggering multiple
; warnings for one *team command in existing data from before this check was
; implemented.
*team "Olly Betts" lefts rights
; Roles should not be quoted.
*team Bob "pics"
*team "Al Jobs
*team
*fix 1 0 0 0
1 2 1 0 0
*end survey
