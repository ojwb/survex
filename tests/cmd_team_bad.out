./cmd_team_bad.svx:3:14: warning: Unknown team role "note"
 *team Wookey note
              ^~~~
./cmd_team_bad.svx:4:32: warning: Unknown team role "tap"
 *team "Olly Betts" instruments tap
                                ^~~
./cmd_team_bad.svx:8:20: warning: Unknown team role "lefts"
 *team "Olly Betts" lefts rights
                    ^~~~~
./cmd_team_bad.svx:10:11: warning: Unknown team role ""
 *team Bob "pics"
           ^
./cmd_team_bad.svx:11:15: warning: Missing "
 *team "Al Jobs
               ^
./cmd_team_bad.svx:12:6: warning: Expecting string field
 *team
      ^

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 2 survey stations, joined by 1 leg.
There are 0 loops.
Total length of survey legs =    1.00m (   1.00m adjusted)
Total plan length of survey legs =    1.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from survey.2 at 0.00m to survey.2 at 0.00m)
North-South range = 1.00m (from survey.2 at 1.00m to survey.1 at 0.00m)
East-West range = 0.00m (from survey.2 at 0.00m to survey.2 at 0.00m)

There were 6 warning(s).
